sdk/core/Azure.Core.TestFramework/README.md

```diff
@@ -380,3 +380,25 @@ For example:
 
 ## Management libraries
 Testing of management libraries uses the Test Framework and should generally be very similar to tests that you write for data plane libraries. There is an intermediate test class that you will likely want to derive from that lives within the management code base - [ManagementRecordedTestBase](https://github.com/Azure/azure-sdk-for-net/blob/babee31b3151e4512ac5a77a55c426c136335fbb/common/ManagementTestShared/ManagementRecordedTestBase.cs). To see examples of Track 2 Management tests using the Test Framework, take a look at the [Storage tests](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/storage/Azure.ResourceManager.Storage/tests/Tests).
+
+## Recording tests on CI
```

Comment:
You should add a note about needing to login and needing permissions to be able to do these operations. Only team members with access to our internal pipelines will succeed in running these.

Reply:
Done